### PR TITLE
[G2M]Tabs - add vertical tabs option + return controlled tab value & onChange handler

### PR DESCRIPTION
**Pull request by hyx131**

### CODE REVIEW

File: src/tab-panels.js [hunk 1]
```diff
@@ -24,18 +36,18 @@ const TabPanels = ({
     ))
 
   return (
-    <>
-      <TabsComponent value={value} onChange={onTabChange}>
+    <div className={TabsProps.orientation === 'vertical' ? classes.vertical : null}>
```
Comment (woozyking) [inline src/tab-panels.js:39]: would this be effectively equivalent?

```suggestion
    <div className={classes[TabsProps.orientation]}>
```

**Review by woozyking — APPROVED** [review]

**Comment by woozyking** [comment]
this should be categorized as one of the "complex" component that may deserve it's own repo such as `lumen-tabs`

File: src/tab-panels.js (hunk 1 above)
Comment (hyx131) [reply src/tab-panels.js:39]: yes it would! 😃

**Review by woozyking — APPROVED** [review]

**Comment by woozyking** [comment]
perhaps update the PR title to reflect key points of updates. if this gets any deeper, merge and start another one to progress further

**Pull request merged**